xml/slurm.xml

```diff
@@ -847,6 +847,14 @@ Description="subgroup" Organization=bavaria</screen>
   upgrade and replace deprecated configuration options before the final restart
   of a service.
  </para>
+ <para>
+  It should be noted that a new major version of Slurm introduces a new version
+  of <literal>libslurm</literal>. Older versions of this library may no longer
```

Comment:
We only use `may` to express permissions. For expressing possibility we use `might`. See https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-vocabulary

```suggestion
  of <literal>libslurm</literal>. Older versions of this library might no longer
```